.env.example

```
@@ -0,0 +1,240 @@
+############
+#FLOWISE_USERNAME=your_username
+#FLOWISE_PASSWORD=your_password
+
+
+############
+# Auth - Configuration for the GoTrue authentication server.
+############
+
+## General
+SITE_URL=http://localhost:3000
+ADDITIONAL_REDIRECT_URLS=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ADDITIONAL_REDIRECT_URLS key should go before the SITE_URL key

(UnorderedKey)